test/file04.ts

```diff
@@ -0,0 +1,22 @@
+import styled from '@emotion/styled';
```

Comment:
File does not have any imports or exports. May need to import or export something.